src/components/AppHeader/AppHeader.vue

```diff
@@ -239,6 +239,9 @@ export default {
       event.preventDefault();
       this.$root.$emit('skip-navigation');
     },
+    getImageUrl() {
```

Comment:
does it make more sense to name these getLogoImage()?

Reply:
getImageUrl seems fine for now.